$NetBSD: patch-meson.build,v 1.1 2024/11/21 15:26:08 hauke Exp $

While NetBSD has libwrap, the meson code doesn't find it. The test
cond still succeeds, so add a branch that does not require the
library.

--- meson.build.orig	2024-11-09 23:29:31.000000000 +0000
+++ meson.build
@@ -1993,7 +1993,11 @@ int main(void) {
 }
 '''
 
-    have_tcpwrap = wrap.found() and cc.links(tcpwrap_code, args: '-lwrap')
+    # *BSD have tcpwrappers support in libc
+    have_tcpwrap = wrap.found() and cc.links(tcpwrap_code)
+    if not have_tcpwrap
+	have_tcpwrap = wrap.found() and cc.links(tcpwrap_code, args: '-lwrap')
+    endif
     if not have_tcpwrap
         have_tcpwrap = cc.links(tcpwrap_code, args: ['-lwrap', '-lnsl'])
     endif
